solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/AddressTableViewCell.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  AdressTableViewCell.swift
+//  DeliveryApp
+//
+//  Created by Michelli Cristina de Paulo Lima on 22/02/22.
+//
+
+import UIKit
+
+class AddressTableViewCell: UITableViewCell {
+
+    static let identifier:String = "AdressTableViewCell"
+    
+    private lazy var stackView:UIStackView = {
```

Comment:
Considere adicionar um espaço sempre na declaração dos types das propriedades.
```suggestion
    private lazy var stackView: UIStackView = {
```